src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -41,6 +46,35 @@ public Drive() {
     }
     /* setting the motors to brake mode */
     drive.setMotorIdleMode(true);
+
+    /* Configure AutoBuilder last */
+    AutoBuilder.configureHolonomic(
+        this.drive::getPose, /* Robot pose supplier */
+        this.drive
+            ::resetOdometry, /* Method to reset odometry (will be called if your auto has a starting pose) */
+        this.drive::getRobotVelocity, /* ChassisSpeeds supplier. MUST BE ROBOT RELATIVE */
+        this.drive
+            ::setChassisSpeeds, /* Method that will drive the robot given ROBOT RELATIVE ChassisSpeeds */
```

Comment:
Are we 100% positive this takes a ROBOT RELATIVE ChassisSpeeds?

Reply:
no

Reply:
I'm decently positive